.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,75 @@
+name: Bug Report
+description: Create a report to help us improve Xournal++.
+labels: bug
+body:
+  - type: dropdown
+    attributes:
+      label: Operating System
+      description: Select the operating system that you use.
+      options:
+        - Windows
+        - Mac OS
+        - Linux
```

```suggestion
        - Windows
        - Mac OS
        - Linux
        - BSDs
```

---

```suggestion
        - Windows
        - Mac OS
        - Linux
        - BSD
```
Perhaps this instead? I am not at all familiar with BSD, so it's quite possible that I am incorrect on this. I'm just wondering if it would look "cleaner" to just write BSD, if possible.

---

I'm fine with `BSD`.